CHANGELOG.md

```diff
@@ -9,6 +9,8 @@
   turned on by default
 - Added a command line provider `config.NewCommandLineProvider()`, which can be used
   to pass configuration parameters through command line.
+- Refactored `Loader.Path()` to `Loader.Paths()`, because we actually look for config
```

Comment:
- `Loader.Path() string` is now `Loader.Paths() []string`, to better reflect that configuration is loaded from multiple directories.